kubernetes/resource_kubernetes_service_v1.go

```diff
@@ -316,6 +316,10 @@ func resourceKubernetesServiceSchemaV1() map[string]*schema.Schema {
 												Type:     schema.TypeString,
 												Computed: true,
 											},
+											"ip_mode": {
+												Type:     schema.TypeString,
+												Computed: true,
```

Comment:
what's the reason for setting it to `Computed: true`?

Also we'll want to add in a `ValidationFunc` since the `ip_mode` can only be two values. This is discussed in the docs [here](https://github.com/hashicorp/terraform-provider-kubernetes/actions/runs/12756331934/job/35554244472)

From docs:

> There are two possible values for .status.loadBalancer.ingress.ipMode: "VIP" and "Proxy". The default value is "VIP" meaning that traffic is delivered to the node with the destination set to the load-balancer's IP and port.